### PR TITLE
fix: add metrics token support

### PR DESCRIPTION
**Pull request by wtrocki**
Metrics guide should follow the same pattern to tell users to copy token url from UI and CLI. 
Alternatively we can give them curl to obtain that value (if somehow they did not copied it before)

Source: https://issues.redhat.com/browse/MGDSTRM-8460

### CODE REVIEW

**Review by sterobin — COMMENTED** [review]
@wtrocki Excellent catch on this! I pushed a commit with some additional tweaks. Hope that's okay, was just easier.

- I revised the text a bit to use the same approach and language as we do with the same info in other docs, for consistency.
- Added the prereq to also be consistent with how we address it in other docs and to also make it clearer.
- Also made the same change in the similar code example that follows it.
- Removed the bit about using curl to obtain the token ID. We already direct them to the UI for everything else they need to obtain, so for simplicity I think it's better IMO to stick with that approach and not introduce additional options for just that one thing. I tried squeezing it in as a note, but just seemed out of place and unnecessary. Open to discussion of course if you disagree.

@AedinC ^^

**Comment by wtrocki** [comment]
Amazing collaboration. This was low effort PR from my side. Now it is very well ready to be merged. Thank you so much for making the change.

Once that is merged all token urls will be consistently provided by UI and CLI. Some example outputs still have it but that is not affecting users in any way

**Comment by sterobin** [comment]
@wtrocki ack

@AedinC Since the related Jira belongs to you, can you have a look and approve if you agree? Just to ensure I'm not stepping on your toes. Once you're good, go ahead with the merge. Thanks.

**Review by AedinC — APPROVED** [review]
LGTM! Thanks @sterobin!

**Comment by wtrocki** [comment]
LGTM

**Pull request merged**
